broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -216,34 +285,68 @@ class ServiceFabrikApiController extends FabrikBaseController {
       });
   }
 
+
   startBackup(req, res) {
+    let backup_started_at;
     req.manager.verifyFeatureSupport('backup');
     const trigger = _.get(req.body, 'trigger', CONST.BACKUP.TRIGGER.ON_DEMAND);
+    let backup_guid, deploymentName;
     return Promise
       .try(() => this.checkQuota(req, trigger))
-      .then(() => {
+      .then(() => Promise.all([utils
+        .uuidV4(),
+        req.manager
+        .findNetworkSegmentIndex(req.params.instance_id)
+        .then(networkIndex => req.manager.getDeploymentName(req.params.instance_id, networkIndex))
+      ]))
+      .spread((guid, deployment) => {
         _.set(req.body, 'trigger', trigger);
-        const bearer = _
-          .chain(req.headers)
-          .get('authorization')
-          .split(' ')
-          .nth(1)
-          .value();
-        return this.fabrik
-          .createOperation('backup', {
-            instance_id: req.params.instance_id,
-            bearer: bearer,
-            arguments: req.body,
-            isOperationSync: true,
-            username: req.user.name,
-            useremail: req.user.email || ''
+        backup_guid = guid;
+        const backup_options = {
+          guid: backup_guid,
+          deployment: deployment,
+          instance_guid: req.params.instance_id,
+          plan_id: req.body.plan_id,
+          service_id: req.body.service_id,
+          context: req.body.context
+        };
+        logger.info('annotate ', req.params.instance_id);
```

Comment:
can this log be more explanatory ?

Reply:
annotateResource prints log statement. So removing this.